devito/interfaces.py

```diff
@@ -4,11 +4,12 @@
 from tools import aligned
 import os
 from signal import signal, SIGABRT, SIGINT, SIGSEGV, SIGTERM
+from tempfile import gettempdir
 import sys
 import atexit
 
 
-__all__ = ['DenseData', 'TimeData', 'PointData']
+__all__ = ['DenseData', 'TimeData', 'PointData', 'MemmapManager', 'SymbolicData']
```

Comment:
I still don't like that these utility classes are exposed in the package. Is there no way to ell sphinx to generate docs for them anyway?